### PR TITLE
fix(xmr): Remove inv8 optimization as in xmr master

### PR DESCRIPTION
**Pull request by ph4r05**
I was notified that Monero changed a BP+ result format in this PR: https://github.com/monero-project/monero/pull/8277

As it affects XMR transaction signing, we have to adapt to the change. Corresponding changes in the Trezor-client code in the XMR wallet are already incorporated in the https://github.com/monero-project/monero/pull/8299

I will send a testing binary once I manage to build it (should be soon).

PR is related to the:
- https://github.com/trezor/trezor-firmware/pull/2232
- https://github.com/trezor/trezor-firmware/pull/2219
- https://github.com/monero-project/monero/pull/8299
- https://github.com/monero-project/monero/pull/8277

### CODE REVIEW

**Comment by ph4r05** [comment]
@matejcik

**Comment by ph4r05** [comment]
Testing binary: https://github.com/ph4r05/monero/releases/tag/v0.17.3.2-dev-tests-u18.04-02

**Comment by prusnak** [comment]
> Testing binary: https://github.com/ph4r05/monero/releases/tag/v0.17.3.2-dev-tests-u18.04-02

Will you please provide the sha256 of the binary?

**Comment by ph4r05** [comment]
> > Testing binary: https://github.com/ph4r05/monero/releases/tag/v0.17.3.2-dev-tests-u18.04-02
> 
> Will you please provide the sha256 of the binary?

@prusnak I've added it to the release notes this time :)

shasum -a256 trezor_tests
f790ad7485cbbad92fbea673a75935028f61d42f64627a7c55f850f6742cf93a

**Comment by ph4r05** [comment]
I've changed testing binary info in the PR (forgot to do that originally)

**Review by matejcik — APPROVED** [review]

**Comment by matejcik** [comment]
waiting for CI run before merging

**Comment by ph4r05** [comment]
@matejcik tests passed https://gitlab.com/satoshilabs/trezor/trezor-firmware/-/jobs/2556760335 :) 🎉

**Comment by matejcik** [comment]
yup \o/

**Comment by ph4r05** [comment]
I will have to make one more PR regarding view tags: https://github.com/monero-project/monero/commit/ea87b30f8907ee11252433811e7a7d0c46758cca 

Some time this week

**Pull request merged**
